content/docs/100-get-started/300-app-lifecycle.mdx

```diff
@@ -11,20 +11,35 @@ Depending on if there is one or many presentations in the playlist, your app’s
 
 ## Single Presentation
 
+<div>
+  <Zoom className="img-light">
+    <img className="img-light" src="/single-presentation-lifecycle-light.excalidraw.png" style={{ maxWidth: 400 }} />
+  </Zoom>
+  <Zoom overlayBgColorStart="rgba(0, 0, 0, 0)" overlayBgColorEnd="rgba(0, 0, 0, 0.95)">
+    <img className="img-dark" src="/single-presentation-lifecycle-dark.excalidraw.png" style={{ maxWidth: 400 }} />
+  </Zoom>
+</div>
```

Comment:
Might be worth creating a component for this but so far I'm only using it on this page.